spec/rspec/core/runner_spec.rb

```diff
@@ -290,6 +290,24 @@ def run_specs
 
             run_specs
           end
+
+          if RSpec::Support::RubyFeatures.supports_exception_cause?
+            it "prevents the DRb error from being listed as the cause of expectation failures" do
+              subclass = Class.new(RSpec::Core::Runner) do
+                include RSpec::Matchers
+
+                def run(err, out)
+                  expect(1).to eq(2)
+                end
+              end
+
+              expect {
+                subclass.run([], StringIO.new, StringIO.new)
```

Comment:
Probably, although this failed w/o my change and passed with it...

Maybe the `before` hook on line 252 takes care of this?